src/feature-viewer.js

```diff
@@ -1,11 +1,16 @@
+const { crossIcon, deleteIcon, bottomArrowIcon } = require('../utils/icons')
+const { dropdownOptions } = require('../utils/data')
+
 function createFeature(sequence, div, options) {
 //        var nxSeq = sequence.startsWith('NX_') ? true : false;
         var self = this;
         // if (!div) var div = window;
         this.events = {
           FEATURE_SELECTED_EVENT: "feature-viewer-position-selected",
             FEATURE_DESELECTED_EVENT: "feature-viewer-position-deselected",
-          ZOOM_EVENT: "feature-viewer-zoom-altered"
+          ZOOM_EVENT: "feature-viewer-zoom-altered",
+          GET_PREDICTIONS_EVENT: "feature-viewer-vep-predictions",
```

Comment:
Is this to support VEP related feature?

Reply:
Yes, this is the event we're calling when the user clicks on the `Get Predictions` button, it retrieves the final variant values added through the popups.